pipelines/README.md

```diff
@@ -175,6 +175,14 @@ The `tekton/gitops-update-pipeline/example-pipelineruns/` contains some examples
 In these examples, notice that there is a template Secret file for the Git credentials that are referenced by different tasks.
 Create an equivalent Secret with appropriate details for your environment, and change the parameter values in the PipelineRun definition to match.
 
+NOTE: If you are using a branch other than `main` for your gitops workflow, you will need to modify the `PipelineRuns` examples under `tekton/gitops-update-pipeline/example-pipelineruns/` to set the `gitRepoBranchBase` parameter to your custom branch name
```

Comment:
```suggestion
> **Note**
> If you are using a branch other than `main` for your gitops workflow, you will need to modify the `PipelineRuns` examples under `tekton/gitops-update-pipeline/example-pipelineruns/` to set the `gitRepoBranchBase` parameter to your custom branch name
```

Looks better:

> **Note**
> If you are using a branch other than `main` for your gitops workflow, you will need to modify the `PipelineRuns` examples under `tekton/gitops-update-pipeline/example-pipelineruns/` to set the `gitRepoBranchBase` parameter to your custom branch name

Reply:
```suggestion
> [!NOTE]
> If you are using a branch other than `main` for your gitops workflow, you will need to modify the `PipelineRuns` examples under `tekton/gitops-update-pipeline/example-pipelineruns/` to set the `gitRepoBranchBase` parameter to your custom branch name
```

@piotrpdev Reading between the lines of the update from 21 July 2023 in the description of [this issue](https://github.com/orgs/community/discussions/16925), my understanding is that this is the more up-to-date syntax for this beta feature, and will replace the older syntax that you suggested. It says that "the old syntax will continue to work for some time", which I read as "we're not going to support the old syntax forever".